kong/plugins/aws-lambda/iam-ecs-credentials.lua

```diff
@@ -114,6 +114,10 @@ local function fetchCredentials()
   return result, nil, result.expiration - ngx_now()
 end
 
+local function fetchRegion()
+  return nil, "Not implemented"
```

Comment:
can't we add this?

Reply:
Sure I have done it, we are not using it in ecs need to deploy to test